core/trino-main/src/test/java/io/trino/execution/TestQueryManagerConfig.java

```diff
@@ -143,6 +144,7 @@ public void testExplicitPropertyMappings()
                 .put("query.max-planning-time", "1h")
                 .put("query.max-cpu-time", "2d")
                 .put("query.reported-rule-stats-limit", "50")
+                .put("query.dispatcher-query-pool-size", "150")
```

Comment:
Make this not a multiple of ten, so that it can't possibly match the default value

Reply:
updated